scripts/install-nbd-kmod.sh

```diff
@@ -10,7 +10,7 @@ KSRC='3.10.0-957.5.1.el7'
 
 # other vars
 TARGET=x86_64
-DISTSITE='http://vault.centos.org/7.6.1810'
+DISTSITE='http://mirror.nsc.liu.se/centos-store/7.6.1810'
```

Comment:
@abejgonzalez, how did you pick this and why Sweden?

Reply:
It's interestingly annoying that the first few mirrors I've tried don't have stuff in `7.6.1810` and only have a readme that says to go to `vault.centos.org` instead... lol.

Reply:
The LInux Kernel Archives has been a fairly reliable mirror in my experience:
https://archive.kernel.org/centos-vault/7.6.1810

Reply:
Offline @davidbiancolin chose this... I don't think there is a large preference for any particular mirror

Reply:
There were three mirrors listed on centos.vault.org. I was getting piss poor networking performance on archive.kernel.org. ~10s kB/s from my manager instance. For a 96M file. So then i tried the one listed here.